debugview/plugin.cpp

```diff
@@ -288,6 +314,7 @@ class debugview : public threadloop {
         mouse_velocity = mouse_velocity * 0.95;
 
         load_camera_images();
+        load_rgb_depth();
```

Comment:
What if both `cam_type` and `rgb_depth_type` are getting published at the same time?
Seems to me that sequentially running these two functions overrides whatever loaded from `cam_type`.

Reply:
You're right. I'll make changes accordingly.